docs/admin/engines/settings.rst

```diff
@@ -219,6 +219,26 @@ Communication with search engines.
 ``max_redirects`` :
   30 by default. Maximum redirect before it is an error.
 
+``categories_as_tabs:``
```

Comment:
No.

Reply:
I asked because a conjunction (_x_as_y_) in a name of an option is not common.

Reply:
`categories_as_tabs` is way clearer than `category_tabs` because it explains that you should list categories there and these categories function as tabs. `category_tabs` does not communicate any of that. "Category tabs" could be something completely different than categories.